runsc/boot/filter/config.go

```diff
@@ -44,7 +40,7 @@ var allowedSyscalls = seccomp.SyscallRules{
 	},
 	syscall.SYS_CLOSE:         {},
 	syscall.SYS_DUP:           {},
-	syscall.SYS_DUP2:          {},
+	syscall.SYS_DUP3:          {},
```

Comment:
Does this need to change? I don't see dup2 => dup3 in this change.

Reply:
The change is committed in a previous commit(a25a976) from @lubinszARM. I think he may missed some changes in this file.

Reply:
Have collected all the related changes to this patch.